tools/Crank/run-benchmarks.ps1

```diff
@@ -64,27 +64,28 @@ $crankConfigPath = Join-Path `
 
 $isLinuxApp = $CrankAgentVm -match '\blinux\b'
 
-$functionAppRootPath = if ($isLinuxApp) { "/home/$UserName/FunctionApps" } else { 'C:\FunctionApps' }
-$functionAppPath = Join-Path `
```

Comment:
Join-Path replaces all the `/` and `\` with whatever is appropriate on the OS running this script, so invoking this on Windows against a Linux VM leads to the function returning 404 because of the wrong app content path.